src/lib.rs

```diff
@@ -14,19 +14,193 @@ use rustty::CellAccessor;
 type Cell = (usize, usize);
 type CellSet = HashSet<Cell>;
 
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use num::PrimInt;
+
+    const EXPECTED_1082_BITS: [bool; 16] = [
+        false, true, false, true, false, false, false, false,
+        true, true, true, false, false, false, false, false];
+    // 1802 = 10 + 7*(2^8), so writing in little endian byte order but writing
+    // the bits within each byte with MSB on the left, we have
+    // [00001010][00000111].
+    // As a BitVec, we want 01010000 11100000.
+
+    #[test]
+    fn test_bitvec_bit_order() {
```

Comment:
Since these are all already in the test module, do we really need to prefix `test_` to their name? Also, can we somehow move all the test code into its own file?

Reply:
> Also, can we somehow move all the test code into its own file?

Interestingly, that is [not Rust's style](https://doc.rust-lang.org/book/second-edition/ch11-03-test-organization.html).

Reply:
Ah, only integration tests go into `tests/` and in their own files. Alright, cool.

Reply:
I used the `test_` prefix to avoid name collisions since we have

```rust
use super::*
```

Reply:
Ah, I see, okay, agreed :)